coordinator/README.md

```diff
@@ -0,0 +1,33 @@
+# Contrast Coordinator
+
+The Coordinator is the central service in a Contrast deployment.
+It exposes an API to set manifests and to get the history of manifests, and it verifies workloads according to the current manifest.
+The below diagram shows how the Coordinator packages interact in the enterprise version.
+
+```mermaid
+graph TB
+    subgraph Coordinator
+            userapi -->|update state in<br/>reset state in| authority
+            meshapi -->|get state from| authority
+            enterprise/recovery -->|reset state in| authority
+            authority -->|manage latest state in| history
+            history -->|store data by hash<br/>atomically update state| store
+
+    end
+
+    subgraph Workload
+        initializer -->|NewMeshCert| meshapi
+        main
+    end
+
+    subgraph Kubernetes
+        Workload
+        Coordinator
+        ConfigMap
+    end
+
+
+    store -->|store data in<br/>receive data from| ConfigMap
+
+    user -->|GetManifests<br/>SetManifest<br>Recover| userapi
```

Comment:
maybe we could add a connection from another coordinator to enterprise/recovery, just so it is easier to understand what is happening there.

Reply:
Added a second coordinator.